src/rajAJun20/CodingTest10.java

```diff
@@ -0,0 +1,33 @@
+/*Print minimum rotation required to compare both strings equality.
+Example 2  : 
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3*/
+
+package rajAJun20;
+
+public class CodingTest10 {
+	void findNumbersOfRotation(String str1,String str2) {
+		int count = 0;
+		if(str1.length()!=str2.length()){
+			System.out.println("Given strings are not identical..!!");
+		}else {	
+			while(!str1.equals(str2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!str1.equals(str2) && count< str1.length)